plugins/datadog-entity-sync-node/src/services/DatadogServiceFromEntitySync.ts

```diff
@@ -173,8 +176,8 @@ export class DatadogServiceFromEntitySync<
   protected serialize(
     entity: Entity,
     _preload?: PreloadedData,
-  ): DatadogEntityDefinition {
-    return defaultComponentSerializer(entity);
+  ): DatadogEntityDefinition | string {
+    return JSON.stringify(defaultEntitySerializer(entity));
```

Comment:
```suggestion
  ): DatadogEntityDefinition | Entity {
    return defaultEntitySerializer(entity);
```
See above. Could the interface be this? Where is it expecting either the datadog definition or an Entity? I think converting it to a string here is confusing and hurts the type safety.